### PR TITLE
Graphicsmagick 1.3.20 was missing despite being in TS

### PR DESCRIPTION
**Pull request by erasche**
Makes me wonder if I should check the TS for other things not mirrored in this repository...

### CODE REVIEW

File: packages/package_graphicsmagick_1_3_20/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<tool_dependency>
+    <package name="graphicsmagick" version="1.3.20">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">http://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/1.3.20/GraphicsMagick-1.3.20.tar.gz</action>
+                <action type="shell_command">mkdir $INSTALL_DIR/gm</action>
+                <action type="shell_command">./configure --prefix=$INSTALL_DIR/gm/ --enable-shared=yes</action>
+                <action type="shell_command">make</action>
+                <action type="shell_command">make install</action>
```
Comment (bgruening) [inline packages/package_graphicsmagick_1_3_20/tool_dependencies.xml]: use `<action type="make_install">` here.

Comment (erasche) [inline packages/package_graphicsmagick_1_3_20/tool_dependencies.xml]: done

File: packages/package_graphicsmagick_1_3_20/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+description: Contains a tool dependency definition that downloads and compiles version
+  1.3.18 of GraphicsMagick.
+long_description: |
+  GraphicsMagick is the swiss army knife of image processing and
+  it provides a robust and efficient collection of tools and libraries which support
+  reading, writing, and manipulating an image in over 88 major formats including important
+  formats like DPX, GIF, JPEG, JPEG-2000, PNG, PDF, PNM, and TIFF.
+
+  http://www.graphicsmagick.org/
+name: package_graphicsmagick_1_3
```
Comment (bgruening) [inline packages/package_graphicsmagick_1_3_20/.shed.yml]: 1_3_20 I think.

File: packages/package_graphicsmagick_1_3_20/.shed.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+owner: iuc
+remote_repository_url: https://github.com/galaxyproject/tools-iuc/tree/master/packages/package_graphicsmagick_1_3
+type: tool_dependency_definition
```
Comment (bgruening) [inline packages/package_graphicsmagick_1_3_20/.shed.yml]: Here as well.

File: packages/package_graphicsmagick_1_3_20/.shed.yml (hunk 2 above)
Comment (erasche) [inline packages/package_graphicsmagick_1_3_20/.shed.yml]: OH. Yes.

Comment (erasche) [inline packages/package_graphicsmagick_1_3_20/.shed.yml]: Done both

**Comment by erasche** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://gx.hx42.org//job/IUC-Github/97/
Test (probably) passed

**Comment by erasche** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://gx.hx42.org//job/IUC-Github/98/
Test (probably) passed

**Pull request merged**